### PR TITLE
Hotfix 1.2.3 - Documentation Only

### PR DESCRIPTION
**Pull request by mikewiebe**
### Added
- Download link for Nexus 5000 and Nexus 6000 Open Agent Container (OAC).
- OAC programmability guide links.
- Complete cisco_ace documentation.

### CODE REVIEW

**Comment by chrisvanheuveln** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -576,20 +576,259 @@ Permit or deny Fragments for ACL. Valid values are 'permit-all' and 'deny-all'
...
+| N56xx    | unsupported        | unsupported            |
+| N6k      | unsupported        | unsupported            |
+| N7k      | unsupported        | unsupported            |
+| IOS XR   | unsupported        | unsupported            |
+
+#### <a name="cisco_ace-caveats">Caveats</a>
+
+| Property | Caveat Description |
+|:--------|:-------------|
+| `http_method` | Not supported on N56xx, N6k, N7k |
```
Comment (glennmatthews) [inline README.md]: We already say above that the whole provider is unsupported on these platforms - why call out individual properties as well?

File: README.md [hunk 2]
```diff
@@ -576,20 +576,259 @@ Permit or deny Fragments for ACL. Valid values are 'permit-all' and 'deny-all'
...
+Determines whether the config should be present or not on the device. Valid values are 'present' and 'absent'.
+
+#### Properties
+
+##### `action`
+The action to perform with this ACE. Valid values are keywords `permit` or `deny`.
+
+| Example
+|:--
+| `action => 'permit'`
```
Comment (glennmatthews) [inline README.md]: These examples take an already long provider and make it even longer - why are they needed/useful here when we don't have them for any of the other providers? I can see where they're useful for specific complex properties (dst_addr, packet_length) but for ones where the property takes just a keyword or a boolean, they seem redundant.

File: README.md (hunk 1 above)
Comment (chrisvanheuveln) [inline README.md]: These platforms are coming in a few weeks. Is it hurting anything to have this information here early?

File: README.md (hunk 2 above)
Comment (chrisvanheuveln) [inline README.md]: True, this format isn't there for other providers but one of the comments we got back from the TMEs was that they had difficulty figuring out how to use some of these properties; ie. we have examples in the type definitions but you have to look up each type file or do puppet describe to see that; so this was an attempt to get this documentation moving in a direction that included example usage.

Do we need examples for every property? Maybe not. I added them for completeness. I'd prefer to keep them but we can remove them if you feel strongly about it.

Comment (glennmatthews) [inline README.md]: Just trying to figure how we can keep this document from growing without bounds. 

    For simple properties like this one, I just don't see how:

Valid values are keywords `permit` or `deny`.

    is really more difficult to figure out than:

Valid values are keywords `permit` or `deny`.

| Example
|:--
| `action => 'permit'`

File: README.md (hunk 1 above)
Comment (glennmatthews) [inline README.md]: Just seemed confusing to me without that context.

File: README.md (hunk 2 above)
Comment (mikewiebe) [inline README.md]: @glennmatthews @chrisvanheuveln Can I go ahead with this merge or is there something that is a must change/fix here?

Comment (chrisvanheuveln) [inline README.md]: :+1:

Comment (glennmatthews) [inline README.md]: :+1: with gripes.

**Pull request merged**